app.rb

```diff
@@ -0,0 +1,37 @@
+require 'sinatra/base'
+require './lib/chitter.rb'
+require './lib/database_connection_setup.rb'
+
+class ChitterApp < Sinatra::Base
+  # enable :sessions
+  # configure(:development) { set :session_secret, "something" }
+  get '/' do
+    redirect '/chitter'
+  end
+
```

Comment:
Nice RESTful routing, have you thought about having the chitter/new page to be on the index page?